io/las/LasReader.cpp

```diff
@@ -67,6 +67,7 @@ class invalid_stream : public pdal_error
 
 void LasReader::processOptions(const Options& options)
 {
+    m_skipWktVLR = options.getValueOrDefault<bool>("skip_vlr_wkt", false);
```

Comment:
Do tests pass with this?  Because mine fail, which is why the fix hasn't been merged.